pages/doc/integrations_custom_app_metrics_howto.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Monitor Custom App Metrics
+keywords:
+tags: [integrations]
+sidebar: doc_sidebar
+permalink: integrations_custom_app_metrics_howto.html
+summary: Learn how to ingest and monitor your custom application metrics.
+---
+
+By default, the Tanzu Observability by Wavefront Nozzle provides metrics for monitoring the TAS platform. For monitoring applications that run **on** the platform, the nozzle supports two models:
+  * **Metric Registrar** -- A **pull-based** model based on the **Prometheus metrics format.**
+  * **Wavefront Service Broker** -- A **push-based** model where applications send metrics directly to a Wavefront proxy.
+
+**We recommend the Metric Registrar approach** as a simpler and more predictable pattern, but teams may choose either option. Both have drawbacks, and for some workloads only the push method can be used in practice.
+
+## Push vs Pull Models Generally
```

Comment:
Push vs Pull Models Overview

I think we should either use "in General" or "Overview" for the title of this section. I prefer overview.